frontEnd/tickets-app/src/components/App/App.js

```diff
@@ -0,0 +1,16 @@
+import Header from "../Header/Header";
+import Main from "../Main/Main";
+import Footer from "../Footer/Footer";
+import React, { Fragment } from "react";
+
+const App = (props) => {
```

Comment:
you can emit `props` here because there is no need in that if you don't use it, just write `()`, fix it throughout your app